tasks/GPSR.tex

```diff
@@ -1,7 +1,6 @@
 \section{General Purpose Service Robot}
 \label{test:gpsr}
-Similar to a smart speaker, the robot is asked to execute arbitrary commands requested by an operator.\\
-\textbf{Note:} Due to this task's more open-ended nature, the maximum possible score is higher than the scores of the other \SONE{} tasks.\\
+The robot is asked to understand and execute commands requiring a wide range of different abilities. \\
```

Comment:
Maybe "natural language commands" to clarify, that the commands won't follow a well defined fixed immutable structure.

Reply:
No, GPSR commands for now. If teams score, make it more difficult